classes/local/object_manipulator/checker.php

```diff
@@ -49,15 +49,22 @@ protected function get_query_name() {
         return 'get_check_candidates';
     }
 
+    /**
+     * @return string
+     */
     protected function get_candidates_sql() {
-        $sql = 'SELECT f.contenthash
+        return 'SELECT MAX(f.id) AS fid,
```

Comment:
After grepping the Codebase, the fid is never used, as you have swapped over to o.id as the primary identifier. Is the fid still needed? I could be missing something

Reply:
`fid` is not referenced, I just want to keep it for consistency.